rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1187,67 +1196,80 @@ private void processPegoutsInBatch(
         long currentBlockNumber = rskExecutionBlock.getNumber();
         long nextPegoutCreationBlockNumber = getNextPegoutCreationBlockNumber();
 
-        if (currentBlockNumber >= nextPegoutCreationBlockNumber) {
-            List<ReleaseRequestQueue.Entry> pegoutEntries = pegoutRequests.getEntries();
-            Coin totalPegoutValue = pegoutEntries
-                .stream()
-                .map(ReleaseRequestQueue.Entry::getAmount)
-                .reduce(Coin.ZERO, Coin::add);
-
-            if (wallet.getBalance().isLessThan(totalPegoutValue)) {
-                logger.warn("[processPegoutsInBatch] wallet balance {} is less than the totalPegoutValue {}", wallet.getBalance(), totalPegoutValue);
-                return;
-            }
+        if (isCurrentBlockBelowNextPegoutCreationBlockNumber(currentBlockNumber, nextPegoutCreationBlockNumber)) {
```

Comment:
Not sure, but I find it clearer just using the less than operator here.

Reply:
i think i agree

Reply:
Undo.